compositor_pipeline/src/pipeline/decoder/opus_decoder.rs

```diff
@@ -74,11 +71,9 @@ impl OpusDecoder {
             };
         }
     }
-}
 
-impl From<AudioChannels> for opus::Channels {
-    fn from(value: AudioChannels) -> Self {
-        match value {
+    fn opus_channels(audio_channels: &AudioChannels) -> opus::Channels {
+        match audio_channels {
```

Comment:
Since `AudioChannels` is now defined in `compositor_renderer` I can't impl `From` trait here.

I can:
1. Do it in this way, which is less idiomatic
2. Defind separate `AudioChannels` in the pipeline - IMO too much overhead
3. impl From in `compositor_renderer` - makes `compositor_renderer` depended on `opus` crate

IMO available options are 1 and 3. I don't have a strong preference, but making `compositor_renderer` dependent on `opus` just for this one conversion seems to me like a bad idea.

Reply:
Nevermind, I moved `AudioChannels` to pipeline